canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -22,6 +22,18 @@
 		margin-bottom: $spacing-03;
 	}
 	.properties-checkboxset-container {
+		.checkbox-tooltip-container{
+			display: grid;
+			justify-content: center;
+			align-items: center;
+			grid-template-columns: 1fr 4fr;
+		}
+		.bx--checkbox-label-text{
+			margin-right: 5px;
+		}
+		.tooltip-container{
+			margin-top: 0.8rem;
```

Comment:
Why is this needed?

Reply:
In checkboxset I added this bit spacing to have tooltips aside horizontally with their associated label  
<img width="439" alt="image" src="https://user-images.githubusercontent.com/89609176/174674524-79008e5e-4147-4c57-87e0-5c0d6e29d8d7.png">
<img width="439" alt="image" src="https://user-images.githubusercontent.com/89609176/174674541-7288d883-2cf6-4609-91cb-a4357d481889.png">